### PR TITLE
Js best practices

### PR DESCRIPTION
**Pull request by SekoViper**
### Js best practices
A todo list with crud operations.

### CODE REVIEW

File: modules/todoList.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        <input class="desc" value="${todo.description}" type="text">
+      </form>
+    </label>
+    <div class="edit">
+    <i id='menu' class="menu fa-solid fa-ellipsis-vertical pointer"></i>
+    <i id='del' class="delete fa-regular fa-trash-can pointer hide"></i>
+    </div>
+  </div>
+    <hr>
+    `;
```
Comment (Htetaungkyaw71) [inline modules/todoList.js]: Kindly create a separate function for this html because of clean code reason.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import _ from 'lodash';
```
Comment (Htetaungkyaw71) [inline src/index.js]: Kindly remove unnecessary comment because of clean code reason.

File: modules/todoList.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  updateTodo();
+  interactivity();
+};
+
+// Add a task
+const addTodo = () => {
+  const form = document.getElementById('task-form');
+  const addInput = document.getElementById('todoTextField');
+  addInput.addEventListener('keypress', (event) => {
+    if (event.key === 'Enter' && addInput.value !== '') {
```
Comment (Htetaungkyaw71) [inline modules/todoList.js]: Kindly use ES6 destructuring for this event object. This is for best practice in JavaScript. Eg -  
```
addInput.addEventListener('keypress', ({key}) => {
    if (key === 'Enter' && addInput.value !== '') {
```

File: modules/todoList.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        <input class="checkbox" type="checkbox">
+        <input class="desc" value="${todo.description}" type="text">
+      </form>
+    </label>
+    <div class="edit">
+    <i id='menu' class="menu fa-solid fa-ellipsis-vertical pointer"></i>
+    <i id='del' class="delete fa-regular fa-trash-can pointer hide"></i>
+    </div>
+  </div>
+    <hr>
```
Comment (Htetaungkyaw71) [inline modules/todoList.js]: Kindly use ES6 destructuring for this todo object. This is for best practice in JavaScript. 
Eg - 
```
  todos.forEach(({id,description}) => {
    const content = `
    <div id="${id}" class="label-container">
    <label class="container">
      <form class="task-edit">
        <input class="checkbox" type="checkbox">
        <input class="desc" value="${description}" type="text">
      </form>
    </label>
    <div class="edit">
    <i id='menu' class="menu fa-solid fa-ellipsis-vertical pointer"></i>
    <i id='del' class="delete fa-regular fa-trash-can pointer hide"></i>
    </div>
  </div>
    <hr>
```

**Review by Htetaungkyaw71 — COMMENTED** [review]
# Changes Required ♻️
### Hi SekoViper,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

File: modules/todoList.js (hunk 3 above)
Comment (SekoViper) [reply modules/todoList.js]: Thanks

**Review by Htetaungkyaw71 — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @SekoViper 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Implemented the previous review requests.
✔️ Able to create, delete and edit a task
✔️ Webpack working well

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**